M2 requireJS Config.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var config={
+  map: {
```

Comment:
We wouldn't always do a map in a requirejs-config, so maybe leave that part out? Also, rename to m2 requirejs-config.js.js